helpers/utilsHelper/Unmarshal.go

```diff
@@ -0,0 +1,23 @@
+package utilsHelper
+
+import (
+	"encoding/json"
+
+	"github.com/astaxie/beego/logs"
+	"github.com/udistrital/utils_oas/errorctrl"
+)
+
+func Unmarshal(in string, out interface{}) (outputError map[string]interface{}) {
```

Comment:
Documentar...
```suggestion
// Unmarshal hace tal cosa...
func Unmarshal(in string, out interface{}) (outputError map[string]interface{}) {
```

PD: Si es conveniente, mover a [`formatdata`](https://github.com/udistrital/utils_oas/tree/master/formatdata)